### PR TITLE
Feature2

### PR DESCRIPTION
**Pull request by jayarajac1**
Feature2 - comments

### CODE REVIEW

File: ProductService/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -4,7 +4,8 @@
 using System.Net;
 using System.Net.Http;
 using System.Web.Http;
-
+//
+//
```
Comment (jayarajac) [inline ProductService/Controllers/ValuesController.cs:8]: remove the comments

**Review by jayarajac — CHANGES_REQUESTED** [review]
comments provided

File: ProductService/Controllers/ValuesController.cs (hunk 1 above)
Comment (jayarajac1) [reply ProductService/Controllers/ValuesController.cs:8]: removed

Comment (jayarajac) [reply ProductService/Controllers/ValuesController.cs:8]: fine

**Review by jayarajac — APPROVED** [review]
approved

**Pull request merged**